api/config/v1/config.go

```diff
@@ -35,6 +35,7 @@ type Config struct {
 	Flags     Flags     `json:"flags,omitempty"     yaml:"flags,omitempty"`
 	Resources Resources `json:"resources,omitempty" yaml:"resources,omitempty"`
 	Sharing   Sharing   `json:"sharing,omitempty"   yaml:"sharing,omitempty"`
+	Imex      Imex      `json:"imex,omitempty"      yaml:"imex,omitempty"`
```

Comment:
I added an `Imex` config section here. Does it make sense to also move the domain config file to this section?

Reply:
As discussed in Slack, I agree with this approach